src/libraries/System.Private.CoreLib/src/System/ObjectDisposedException.cs

```diff
@@ -44,6 +45,14 @@ protected ObjectDisposedException(SerializationInfo info, StreamingContext conte
             _objectName = info.GetString("ObjectName");
         }
 
+        [DoesNotReturn]
+        public static void Throw(object instance) =>
+            Throw(instance.GetType());
```

Comment:
```suggestion
            Throw(instance?.GetType());
```

Reply:
But even without the argument being nullable, I agree with @martincostello that if null is passed in, it shouldn't null ref.